integration_tests/e2e/overview/educationAndTrainingTab.cy.ts

```diff
@@ -38,7 +38,47 @@ context('Prisoner Overview page - Education And Training tab', () => {
       educationAndTrainingPage //
         .activeTabIs('Education and training')
         .hasFunctionalSkillsDisplayed()
-        .hasCompletedInPrisonQualificationsDisplayed()
+        .hasCompletedInPrisonQualificationsLast12MonthsDisplayed()
```

Comment:
I don't think I 100% follow this ....
How does this test pass? What stub is the test setting up that makes it display the "courses completed in the last 12 months" table? 
(Same question for the other tests where you've added the assertion `.hasCompletedInPrisonQualificationsLast12MonthsDisplayed()` )